file-formats/csnm/examples/z_aff_example_csnm.csnm.json

```
@@ -0,0 +1,18 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "abapLanguageVersion": "standard",
+    "description": "Example CSN model for ABAP file formats",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "csnFileNames": [
+      {
+        "name": "model_parsed.csn"
+      },
+      {
+        "name": "model_inferred.csn"
+      }
+    ]
+  }
```

Just to make my idea of the other comment more clear:

```suggestion
  "csnFileNames": [
    {
      "name": "model_parsed.csn"
    },
    {
      "name": "model_inferred.csn"
    }
  ]
```